lib/resources/usVerifications.js

```diff
@@ -8,8 +8,13 @@ class USVerifications extends ResourceBase {
     super('us_verifications', config);
   }
 
-  verify (params, callback) {
-    return this._transmit('POST', null, null, params, callback);
+  verify (params, qs, callback) {
+    if (typeof qs === 'function') {
```

Comment:
in what case would the query be a function?

Reply:
oh sorry is this to keep compatibility? for people who are still passing in `callback` as the second argument?

why not just make `qs` the third argument? @dmlittle